canvas_modules/common-canvas/src/common-properties/controls/dropdown/dropdown.scss

```diff
@@ -34,6 +34,7 @@
 		}
 		.bx--select-input {
 			width: inherit; // Override carbon default width: auto
+			background-color: transparent; // Override carbon default background-color: #f4f4f4;
```

Comment:
@nmgokhale Is there a prop we can pass into the Dropdown to get the correct color instead of overriding it?

Reply:
In `Select` control, we are [passing `inline` and `light` props](https://github.com/elyra-ai/canvas/blob/master/canvas_modules/common-canvas/src/common-properties/controls/dropdown/dropdown.jsx#L204). 
But for inline Select control, background color is same for `light` true or false. 

In earlier version, carbon had` background-color: transparent` - 
![Screen Shot 2021-11-12 at 3 17 00 PM](https://user-images.githubusercontent.com/25124000/141594382-096452a3-525b-4e20-9e99-56997ac3f5eb.png)


But in version 10.48.0, carbon changed this `background-color` - 
Overriding this color in CSS. 
![Screen Shot 2021-11-12 at 3 18 10 PM](https://user-images.githubusercontent.com/25124000/141594405-d2a9a368-4bb2-4a09-b75f-beed14d2f92a.png)